src/js/components/events-carousel.js

```diff
@@ -0,0 +1,78 @@
+import { newText, newContainer, newMultimedia } from './helpers'
```

Comment:
Please separate the `imports` section from the rest of the component's code with an extra space